### DISCUSSION TITLE
osu!lazer does not open on GDM/Wayland

### DISCUSSION THREAD

**Discussion by imskyyc** [ppy/osu#17312] [Q&A] [answered]
Hi all, I'm trying to run the most recent osu!lazer AppImage on a GarudaLinux GNOME installation running Wayland. The game runs just fine on Xorg / X11, and the game starts on Wayland, I can hear sounds and some buttons here and there, but the window never actually opens. Hovering over it on the task bar shows a small black box that's nowhere to be seen on any of my monitors. I have seen here and there that Wayland support likely won't be a thing (maybe I'm wrong), but in my instance it makes the game unplayable. Wayland is a personal preference due to running better (in my experience) with my NVIDIA card. Is there at all a fix or workaround for this?

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15625
